Makefile-az.mk

```diff
@@ -315,3 +269,18 @@ az-pprof-enable: ## Enable profiling
 az-pprof: ## Profile
 	kubectl port-forward service/karpenter -n karpenter 8000 &
 	sleep 2 && go tool pprof -http 0.0.0.0:9000 localhost:8000/debug/pprof/heap
+
+az-mkaks-user: az-mkrg ## Create compatible AKS cluster, the way we tell users to
+	hack/deploy/create-cluster.sh $(AZURE_CLUSTER_NAME) $(AZURE_RESOURCE_GROUP)
+
+az-helm-install-snapshot: az-configure-values ## Install Karpenter snapshot release
+	$(eval SNAPSHOT_VERSION=$(shell git rev-parse HEAD)) # guess which, specify explicitly with SNAPSHOT_VERSION=...
```

Comment:
Does this override if already specified? I thought you need a `?=` to not override if set?

Reply:
As is this failed for me when trying to override, but confirmed that `?=` fixed it.

Reply:
I meant "just change the line in Makefile", but I like your suggestion better, fixed